TWMessageBarManager.podspec

```diff
@@ -1,16 +1,16 @@
 Pod::Spec.new do |s|
   s.name         = "TWMessageBarManager"
-  s.version      = "1.7.2"
+  s.version      = "1.7.3"
   s.summary      = "An iOS manager for presenting system-wide notifications via a dropdown message bar."
-  s.homepage     = "https://github.com/terryworona/TWMessageBarManager"
+  s.homepage     = "https://github.com/Backelite/TWMessageBarManager"
 
-  s.screenshot   = "https://raw.github.com/terryworona/TWMessageBarManager/master/Screenshots/main.png"
+  s.screenshot   = "https://raw.github.com/Backelite/TWMessageBarManager/master/Screenshots/main.png"
 
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
   s.author       = { "Terry Worona" => "terryworona@gmail.com" }
   s.source       = { 
-	:git => "https://github.com/terryworona/TWMessageBarManager.git",
-	:tag => "v1.7.2"
+	:git => "https://github.com/Backelite/TWMessageBarManager.git",
```

Comment:
This path is incorrect. 

I wouldn't worry about touching the pod spec, I can update that when a new PR is pulled in.